getting-started-github-graphql-api.md

```diff
@@ -23,15 +23,17 @@ summary = "Everything you need to get started with GitHub's GraphQL API for lear
 
 {{% read_before_2 "This tutorial is part 1 of 3 in this series." "Part 2:" "A complete React with GraphQL Tutorial" "https://www.robinwieruch.de/react-with-graphql-tutorial/" "Part 3:" "A complete React with Apollo and GraphQL Tutorial" "https://www.robinwieruch.de/react-graphql-apollo-tutorial/" %}}
 
-Often it is easier to learn something new by learning it step by step. When you start learning about GraphQL in JavaScript, you can dive into the client-side and server-side of an application at the same time. However, this might not be the best approach of learning about GraphQL, because now you have to deal with two environments instead of one. So the step by step mentality would be difficult to apply. That's why I encourage GraphQL newcomers to start with the client-side usage of GraphQL by consuming a third-party GraphQL API before diving into the server-side of building your own GraphQL server. {{% a_blank "GitHub" "https://github.com" %}} is one of the major adopters of GraphQL who managed to {{% a_blank "release" "https://githubengineering.com/the-github-graphql-api" %}} a public GraphQL API {{% a_blank "(official documentation)" "https://developer.github.com/v4" %}}. Their API is quite popular among developers who learn about GraphQL, because most developers are already familiar with their platform to host their projects and source code.
+Step-by-step is often the easiest way learn something new, so it's fortunate that learning GraphQL in JavaScript teaches both the client and the server-side of an application. Seeing both sides of the web transactions is useful, but the catch is you have to learn two environments. The step-by-step mentality can be difficult to apply here, so I encourage newcomers to start client-side, by consuming a third-party GraphQL API, before the server side, which is using a GraphQL server. 
```

Comment:
"... so I encourage newcomers to start client-side, by consuming a third-party GraphQL API, before the server side, which is using a GraphQL server." to "... so I encourage newcomers to start with a client-side application, by consuming a third-party GraphQL API, before diving into the server side, which is using a GraphQL server."